packages/subscriber/src/components/add-form/index.tsx

```diff
@@ -492,6 +499,16 @@ export const AddSubscriberForm: FunctionComponent< Props > = ( props ) => {
 
 					{ renderEmptyFormValidationMsg() }
 
+					{ newsletterCategoriesData?.enabled &&
```

Comment:
We're only showing this section if you have newsletter categories. But we might change this in future and gray out the toggle, so that users can learn about categories.